helm-charts/0.25.4/charts/safe/templates/Identity.yaml

```diff
@@ -8,20 +8,20 @@
 # >/'  SPDX-License-Identifier: BSD-2-Clause
 # */
 
+{{- if .Values.global.useClusterSpiffeIds }}
```

Comment:
For cases that the operator wants to create ClusterSPIFFEIds off-cycle.

(I am, again, looking at you OpenShift).